### ISSUE TITLE
test pre-trained models（256x256）， result is not  right， what's wrong？

### ISSUE THREAD

**Issue by ToBigboss** [yunjey/stargan#102]
code is：
model = Generator()
model = model.cuda()
model.load_state_dict(torch.load("./stargan_celeba_256/models/200000-G.ckpt"))
model = model.eval()

pilImg = Image.open("256x256.jpg")
npImg  = np.array(pilImg)
example_img = torch.from_numpy(npImg).float().unsqueeze(0)
print(example_img.shape)
example_img = example_img.permute(0, 3, 1, 2)
print(example_img.shape)
print("======RGB data [0, 255]======")
print(example_img)
example_img = example_img.div(255)
example_img = example_img.sub(0.5)
example_img = example_img.div(0.5)
example_img = example_img.cuda()

example_c   = torch.cuda.FloatTensor([[-1,-1,1,-1,-1]])

imgGen = model(example_img, example_c)
save_image(imgGen, "./%s.png" % 100, normalize=True)

result is：
![100](https://user-images.githubusercontent.com/40013733/63107066-d7622f80-bfb6-11e9-895e-f3333018c903.png)

**Comment by rex-yue-wu** [comment]
you need to denormalize the image. Check `solver.denorm()`

**Comment by raikuma** [comment]
Disable this line.
```
model = model.eval().
```
It works for me.

**Comment by ujjawalcse** [comment]
Have you resolved the issue @ToBigboss by using these two methods given by @rex-yue-wu and @raikuma ?
I tested the solver.denorm with the line 'model.eval()' commented but result didn't improve.

**Comment by 07hyx06** [comment]
> Disable this line.
> 
> ```
> model = model.eval()
> ```
> 
> It works for me.

Hi! What about your generated image quality?I got very ugly outputs when i loaded pretrained model 200000-G

![tmp6ah556n2_wps图片](https://user-images.githubusercontent.com/52267384/71640304-98f9b280-2cc2-11ea-9ddb-873a32f98c09.jpg)

**Comment by despoisj** [comment]
Has someone investigated this .eval() issue. Is it an issue with the InstanceNorm2D layer?
